library/catalog/views.py

```diff
@@ -148,13 +148,23 @@ def registration(request):
     if request.method == 'POST':
         form = RegistrationForm(request.POST)
         if form.is_valid():
-            user = MongoUser(email=form.cleaned_data.get('email'))
-            user.first_name = form.cleaned_data.get('firstname')
-            user.last_name = form.cleaned_data.get('lastname')
-            user.username = form.cleaned_data.get('login')
-            user.password = form.cleaned_data.get('password')
+            first_name = form.cleaned_data.get('firstname')
```

Comment:
I guess, that this is a better approach:
``username = form.cleaned_data.get('login')
password = form.cleaned_data.get('password')

user = MongoUser(email=form.cleaned_data.get('email'))
user.first_name = form.cleaned_data.get('firstname')
user.last_name = form.cleaned_data.get('lastname')
user.username = username
user.password = password